tools/docker/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV PYTHONUNBUFFERED 1
 ENV PROJECT_HOME /home/patchwork/patchwork
 ENV DJANGO_SETTINGS_MODULE patchwork.settings.dev
 
-RUN groupadd --gid=$GID patchwork && \
+RUN groupadd -o --gid=$GID patchwork && \
```

Comment:
Ah, I hadn't seen this PR. I've just removed all of this logic since it's no longer necessary from what I can tell. I'll revert it in favour of this though.